src/main/java/com/google/sps/servlets/utils/Statistics.java

```diff
@@ -70,8 +70,8 @@ public double getAverageScore() {
    */
   public Statistics(List<UserComment> userCommentList, int topNComments) {
     sentimentBucketList = categorizeToBucketList(userCommentList, topNComments);
-    averageScore = getAverageValue(userCommentList, "score");
-    averageMagnitude = getAverageValue(userCommentList, "Magnitude");
+    averageScore = Math.round(getAverageValue(userCommentList, "score") * 100) / 100.0;
+    averageMagnitude = Math.round(getAverageValue(userCommentList, "Magnitude") * 100) / 100.0;
```

Comment:
I am not sure if this entirely works, as this is still a double? How do you plan to display it? If you are going to cut in the .js file then maybe it is just easier to do so in the frontend?

Reply:
This still works with all tests passed (since we divide /100.0).But I made the change to revert it back and ask molly to handle it in the front end instead